mlflow/metrics/genai/base.py

```diff
@@ -72,17 +72,23 @@ def _format_grading_context(self):
         else:
             return self.grading_context
 
-    def __str__(self) -> str:
+    def print(self, include_input: bool = True) -> str:
```

Comment:
Is there a reason we are using `include_input` in example rather than just not passing in input? Basically making input as optional here? Sorry maybe I missed this decision :(